ISISComputingGroup/EPICS-ioc
# Ticket3353 gem sample changer fix

### Description of work

Now checks `POSN:SP` against current `POSN` and does not process if they match.

### To test

Fixes 3353, to test run python tests which should pass, includes a test to make sure that a record is not processed if `POSN:SP` matches `POSN`

### Acceptance criteria

Sample changer does not move when `POSN:SP` is set to the current `POSN`

---

#### Code Review

    - [PV naming](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming).
    - [Disable records](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Disable-records)
    - [Record simulation](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Record-Simulation)
    - [Finishing touches](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/IOC-Finishing-Touches)

### Functional Tests

- IOC responds correctly in:

### Final steps
